probe-rs/src/probe.rs

```diff
@@ -249,6 +249,7 @@ impl Probe {
     /// Get a list of all debug probes found.
     /// This can be used to select the debug probe which
     /// should be used.
+    #[tracing::instrument(skip_all)]
```

Comment:
I think the `skip_all` doesn't do anything here, since the function doesn't have any arguments.